lineage/Analyze.py

```diff
@@ -225,3 +226,10 @@ def accuracyG(tHMMobj, all_states):
         acuracy_holder.append(100 * acc)
 
     return acuracy_holder
+
+
+def kl_divergence(p, q):
```

Comment:
Why not use [`scipy.stats.entropy`](https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.entropy.html)?

Reply:
Still wondering about this.